lib/class-wp-rest-swagger-controller.php

```diff
@@ -247,18 +247,20 @@ function ($matches) use (&$defaultidParams){
 						}
 						if(!empty($pdetails['type'])){
 							if($pdetails['type']=='array'){
-								$parameter['type']=$pdetails['type'];
-								$parameter['items']=array(
-									'type'=> $pdetails['items']['type'],
-									'properties'=> $pdetails['items']['properties']
-								);
+								if($pdetails['items']) {
+									$parameter['items']=$pdetails['items'];
+								} else {
+									$parameter['items']=array('type'=>'string');
+								}
 							}elseif($pdetails['type']=='object'){
 								$parameter['type']=$pdetails['type'];
-								$parameter['items']=array(
-									'type'=> $pdetails['items']['type'],
-									'properties'=> $pdetails['items']['properties']
-								);
-							
+								if($pdetails['items']) {
+									$parameter['items']=$pdetails['items'];
+								}else if ($pdetails['properties']) {
+									$parameter['properties']=$pdetails['properties']['rendered'];
```

Comment:
Does this make sense to you? I see properties.raw and properties.rendered on some woocommerce endpoints. I just decided to keep the rendered as there appears to be more information. Works for my cases.